cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -273,3 +253,25 @@ func StartControllers(s *options.CloudControllerManagerServer, kubeconfig *restc
 
 	select {}
 }
+
+func startHTTP(s *options.CloudControllerManagerServer) {
```

Comment:
@wackxu I think your motivation is good. However, I can't see any real benefit from this refactor?

Reply:
As we can see some other module that have the similar code ,this has been separated from the original function such as kubernetes\plugin\cmd\kube-scheduler\app\server.go. I think that the piece of code is too long and it has independent perfection function. it will be more readable and reasonable by refactoring the code and it more meets the programming standards。

Reply:
I am not sure if different components will share the same startup.